codec/src/main/java/io/netty/handler/codec/ConvertibleHeaders.java

```diff
@@ -90,13 +90,6 @@
     List<ConvertedType> getAllAndRemoveAndConvert(UnconvertedType name);
 
     /**
-     * Invokes {@link Headers#entries()} and lazily does a conversion on the results as they are accessed
```

Comment:
@nmittler - Lets continue on the main thread.  I have posted https://github.com/netty/netty/pull/3681#issuecomment-95298163 which is related to this discussion too.